static/scss/email-composition-dialog.scss

```diff
@@ -1,67 +1,125 @@
-.email-composition-dialog {
-  .email-composition-contents {
-    display: flex;
-    flex-direction: column;
+.email-composition-dialog-wrapper {
```

Comment:
it was main pr, 
without this setting, when user select the automatic email, ui will be damage .